seatunnel-core/seatunnel-flink-starter/src/main/java/org/apache/seatunnel/core/starter/args/FlinkCommandArgs.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.core.starter.args;
+
+import org.apache.seatunnel.common.config.DeployMode;
+import org.apache.seatunnel.core.starter.command.AbstractCommandArgs;
+import org.apache.seatunnel.core.starter.config.ApiType;
+import org.apache.seatunnel.core.starter.config.EngineType;
+import org.apache.seatunnel.core.starter.config.FlinkRunMode;
+
+import com.beust.jcommander.IStringConverter;
+import com.beust.jcommander.Parameter;
+
+import java.util.List;
+
+public class FlinkCommandArgs extends AbstractCommandArgs {
+
+    @Parameter(names = {"-r", "--run-mode"},
+        converter = RunModeConverter.class,
+        description = "job run mode, run or run-application")
+    private FlinkRunMode runMode = FlinkRunMode.RUN;
+
+    @Parameter(names = {"-api", "--api-type"},
+        converter = ApiTypeConverter.class,
+        description = "Api type, engine or seatunnel")
+    private ApiType apiType = ApiType.ENGINE_API;
```

Comment:
We can remove this parameter, in our new module we don't need this.

Reply:
thx~ done